src/api/app/components/reports_notice_component.html.haml

```diff
@@ -0,0 +1,6 @@
+.text-warning
+  %i.fas.fa-flag
+  - if by_user
+    You reported this #{reportable.class.name.downcase}.
```

Comment:
Maybe we can push this `reportable.class.name.downcase` into the `Report` class, as we use it in several places now.